webrender/res/ps_text_run.glsl

```diff
@@ -70,15 +70,14 @@ void main(void) {
     //note: the blend mode is not compatible with clipping
     oFragColor = texture(sColor0, tc);
 #else
-    float alpha = texture(sColor0, tc).a;
+    vec4 color = texture(sColor0, tc) * vColor;
 #ifdef WR_FEATURE_TRANSFORM
     float a = 0.0;
     init_transform_fs(vLocalPos, a);
-    alpha *= a;
+    color.a *= a;
 #endif
-    vec4 color = vColor;
```

Comment:
This code path used to ignore `sColor0` texture fetch, and now it does not. Is this change intentional?

Reply:
It used to fetch just the ```a``` component for non-subpixel fonts. Now, it fetches and scales the color by the entire ```rgba``` of the texture. This is necessary to support full color glyphs. Normal alpha glyphs that go through this path always have ```(1,1,1)``` in their ```rgb``` components, so it all works the same for those fonts (I verified this visually on all platforms).